libs/ledger-live-common/src/families/avalanchepchain/js-buildTransaction.ts

```diff
@@ -0,0 +1,43 @@
+import type { Transaction } from "./types";
+import { BN } from "avalanche";
```

Comment:
Is there any particular reason we are using avalanche BN and not BigNumber.
I guess it's fine if it's stay on the scope of js-buildTransaction tho. but yeah probably some compatibility issue to build the tx

Reply:
Right, the `buildAddDelegatorTx` function expects `BN` not `BigNumber`, so we build the transaction with `BN`.